### ISSUE TITLE
Mastodon: Automatically enable prefers-reduced-motion

### ISSUE THREAD

**Issue by rugk** [PrivacyDragon/FOSS-accessibility-hub#1]
See https://github.com/tootsuite/mastodon/issues/10975

**Comment by PrivacyDragon** [comment]
Am I correct when I think this needs some CSS to get fixed?
If so, I can add a label 'css'.

**Comment by rugk** [comment]
Yeah, it looks like a CSS-only change. Feel free to comment there/contribute it in the upstream issue.
